config/rubocop.yml

```diff
@@ -19,15 +19,14 @@ Style/StringLiterals:
 Metrics/LineLength:
   Max: 100
```

Comment:
I assume your first response below mine came in before you read my message but just to make sure: yes additional excludes are fine